### ISSUE TITLE
✔️ 🚀 Feature Request: When using elementor padding and margin values separately, replace "0" with something unset instead

### ISSUE THREAD

**Issue by Nazrinn** [elementor/elementor#14842]
## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
Allow more flexibility and performance improvement in CSS and custom CSS.

Currently, when setting a custom margin/padding on elements in Elementor, it outputs CSS as such:
```
.elementor-447 .elementor-element.elementor-element-22b158b {
    padding: 140px 0px 80px 0px;
}
```

However, if I target the same element in my code, to, let's say, give global padding (without a class) like so (like for sections):
```
.elementor-section:not(.elementor-inner-section) {
    padding-left: clamp(20px, 6.15vw, 50px) !important;
    padding-right: clamp(20px, 6.15vw, 50px) !important;
}
```

Suddenly because the original Elementor outputs says `0` where padding left and right is, I am forced to use `!important` to make my custom CSS work. This is incredibly frustrating because when it comes to overriding my global padding in any way, I am forced to use `!important` as well or mess up with the cascading selector, which decreases page rendering performance.

**Describe the solution you'd like**
Simply make Elementor not set unset value to `0` by default in the padding/margin text field, and instead output something like this:
```
.elementor-447 .elementor-element.elementor-element-22b158b {
    padding-top: 140px;
    padding-bottom: 80px;
}
```

Like this, the `padding-left` and `padding-right` stay unset unless explicitly specified, like it should be.

**Describe alternatives you've considered**
Messing with the CSS. It works, but of course, it doesn't really solve the messy code problem.

**Additional context**
[Screenshot of the issue](https://i.gyazo.com/348c80d6b368f5b8c3cadeee3a77402c.jpg)

**Comment by squadcar** [comment]
Came here to say the same thing, this has been an annoyance since first using Elementor. As @Nazrinn mentioned, there's workarounds, but considering you can delete the padding value when using locked values, you should be able to when they are separately input. As of now, deleting a value just sets it to 0. It should unset the value instead of forcing a value of 0.

**Comment by rami-elementor** [comment]
Hi @Nazrinn @squadcar ,

We understand your frustration. Due to higher specificity, using `!important` can be annoying. Yet, replacing existing CSS values have the potential to break existing websites. So we need to consider this very carefully.

You are requesting to replace the shorthand:

```
.elementor-447 .elementor-element.elementor-element-22b158b {
    padding: 140px 0px 80px 0px;
}
```

with:

```
.elementor-447 .elementor-element.elementor-element-22b158b {
    padding-top: 140px;
    padding-bottom: 80px;
}
```

This will increase the CSS size.

Have you tried using different selectors to target all the elements? Looks like `.elementor-section:not(.elementor-inner-section)` has a lower specificity.

**Comment by Nazrinn** [comment]
I'll try that next time if I come across it. So far with the new padding system used by Elementor flexbox containers, this is much less of an issue!

The separation of padding/margin attribute would still be nice though. `unset =/= 0`

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that a solution to this feature request was released with Elementor **v3.10**! 🥳 

You can now use Custom Units on many fields, including dimension fields like margin and padding, allowing you to add CSS functions like clamp() and keywords like unset to each field without needing to use custom CSS.

Check the release post [here](https://elementor.com/blog/new-nested-elements-tabs-custom-units/).

✔️ Feel free to check it out and update your plugin to the new version!

Cheers 🥂
